### PR TITLE
MC-10390 : Draft of the Azure API doc for what is actually in place.

### PR DESCRIPTION
**Pull request by fyrnaga**
This branch is to add documentation on the Azure API plugin for the instances.

### CODE REVIEW

File: source/includes/azure/_instances.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/azure/example/instances/subscriptions/subscriptionId/resourceGroups/cmc-example/providers/Microsoft.Compute/virtualMachines/example-small-server?operation=resize"
+
+# Request example:
+```
+
+```json
+{
+  "region" : "canadacentral",
+  "machineType": "Standard_B1ls"
```
Comment (pdube) [inline source/includes/azure/_instances.md:213]: Can the region be changed?

Comment (fyrnaga) [reply source/includes/azure/_instances.md:213]: No. There is an item opened to correct that in the backlog

Comment (pdube) [reply source/includes/azure/_instances.md:213]: Got it

**Review by pdube — APPROVED** [review]
LGTM 👍

File: source/includes/azure/_instances.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances</code>
+
+Retrieve a list of all instances in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the instance. This is a canonized it from azure which is the form of /subscriptions/:subscriptionid/resourceGroups/:resourcegroup/providers/Microsoft.Compute/virtualMachines/:instanceName
```
Comment (franzpgarcia) [inline source/includes/azure/_instances.md]: `canonized it`?

File: source/includes/azure/_instances.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances/:id</code>
+
+Retrieve an instance in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the instance. This is a canonized it from azure which is the form of /subscriptions/${subscriptionid}/resourceGroups/${resourcegroup}/providers/Microsoft.Compute/virtualMachines/${instanceName}
```
Comment (franzpgarcia) [inline source/includes/azure/_instances.md]: same.

**Review by franzpgarcia — APPROVED** [review]

File: source/includes/azure/_instances.md (hunk 3 above)
Comment (fyrnaga) [reply source/includes/azure/_instances.md]: will correct

File: source/includes/azure/_instances.md (hunk 2 above)
Comment (fyrnaga) [reply source/includes/azure/_instances.md]: will correct

Comment (fyrnaga) [reply source/includes/azure/_instances.md]: corrected

File: source/includes/azure/_instances.md (hunk 3 above)
Comment (fyrnaga) [reply source/includes/azure/_instances.md]: corrected

**Pull request merged**